HealthMonitoring.SelfHost/HealthMonitoring.SelfHost.csproj

```diff
@@ -15,6 +15,20 @@
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
+    <Install>true</Install>
```

Comment:
I dont know what generate this changes
IMHO this changes not needed here, but i can be wrong due to previous line